rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -64,18 +64,18 @@ predicate trivialPat(Pat p) { p instanceof WildcardPat or p instanceof IdentPat
 class AsmExprTree extends LeafTree instanceof AsmExpr { }
 
 class AwaitExprTree extends StandardPostOrderTree instanceof AwaitExpr {
-  override ControlFlowTree getChildNode(int i) { i = 0 and result = super.getExpr() }
+  override AstNode getChildNode(int i) { i = 0 and result = super.getExpr() }
```

Comment:
Why these changes? I don't think they're bad, just wondering why they were made in a commit with title `Rust: Add more CFG tests`.

Reply:
Without this change we do not get the dead-end inconsistency for the `while` loop test, because `WhileExpr` is not (yet) a `ControlFlowTree`.